legacy/core/src/main/java/com/fsck/k9/preferences/Settings.java

```diff
@@ -33,7 +33,7 @@ class Settings {
      *
      * @see SettingsExporter
      */
-    public static final int VERSION = 100;
+    public static final int VERSION = 101;
```

Comment:
This is the version for settings export. If we want the value in the settings file, an entry has to be added to `GeneralSettingsDescriptions`.

Reply:
The settings need to be improved, it's unclear which component belongs to what functionality and what defines that a setting is importable/exportable.